l10n_es_facturae/country.py

```diff
@@ -27,8 +28,26 @@
 class Country(osv.osv):
     _inherit = 'res.country'
     _columns = {
-        'code_3166': fields.char('Country Code', size=3,help='The ISO 3166 country code in three chars.', required=True),
+        'code_3166': fields.char('Country Code', size=3,
+            help='The ISO 3166 country code in three chars.', required=True),
     }
+
+    def _auto_init(self, cr, context=None):
+        res = super(Country, self)._auto_init(cr, context=context)
+        cr.execute("select count(*) from pg_class as c inner join pg_attribute as a on a.attrelid = c.oid where a.attname = 'code_3166' and c.relkind = 'r' and c.relname = 'res_country'")
```

Comment:
Esto en un lugar de hacerlo en el _auto_init, sería más limpio hacerlo como un script de migración, que sólo se ejecuta si `version is None`, que es cuando lo instalas por primera vez. Así, sólo se ejecutará esa vez, y no cada vez que arranques el servidor.